_assets/js/search.js.liquid

```diff
@@ -7,19 +7,6 @@
 //= require 'vendor/jquery.lunr.search'
 
 $(document).ready(function(){
-    $('.search-input').focus(function() {
```

Comment:
Have you rewritten this somewhere else? This is the function that loads the search index